spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -0,0 +1,252 @@
+describe ManageIQ::Providers::Vmware::CloudManager::Refresher do
+  before(:each) do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(
+      :ems_vmware_cloud,
+      :zone     => zone,
+      :hostname => 'localhost',
```

Comment:
I used app secrets in the latest commit. Please note that I also changed the initialisation (`before`) to be more similar to other providers.